code/lib/cli/src/initiate.ts

```diff
@@ -239,18 +245,103 @@ const projectTypeInquirer = async (
   process.exit(0);
 };
 
-async function doInitiate(
-  options: CommandOptions,
-  pkg: PackageJson
-): Promise<
+const scaffoldProject = async ({
+  packageManager,
+  skipInstall,
+}: {
+  packageManager: JsPackageManager;
+  skipInstall?: boolean;
+}): InitiateResult => {
+  logger.log();
+  logger.log(
+    chalk.yellow(
+      dedent`
+        We've detected that your project is empty. We will scaffold a project for you with a pre-configured storybook setup.
+      `
+    )
+  );
+  logger.log();
+  const templateEntries = Object.entries(baseTemplates);
+  const result = await prompts(
+    {
+      type: 'select',
+      message: '🌈 Select a template',
+      name: 'template',
+      choices: templateEntries.map(([key, value]) => ({
+        title: key,
+        value: { key, projectType: value.projectType },
+      })),
+    },
+    {
+      onCancel: () => {
+        logger.log('Command cancelled by the user. Exiting...');
+        process.exit(1);
+      },
+    }
+  );
+
+  const logSuccessfulBootstrapMessage = () => {
+    logger.log(
+      boxen(
+        dedent`
+            You have successfully bootstrapped a Storybook for ${template.key} project! 🎉
+            ${
+              skipInstall
+                ? 'Please install your package.json dependencies manually before continuing.'
+                : ''
+            }
+            To run Storybook manually, run ${packageManager.getRunStorybookCommand()}. CTRL+C to stop.
+            
+            Wanna know more about Storybook? Check out ${chalk.cyan('https://storybook.js.org/')}
+            Having trouble or want to chat? Join us at ${chalk.cyan(
+              'https://discord.gg/storybook/'
+            )}
+          `,
+        { borderStyle: 'round', padding: 1, borderColor: '#F1618C' }
+      )
+    );
+  };
+
+  const template = result.template as { key: string; projectType: ProjectType };
+
+  await sandbox({
+    filterValue: template.key,
+    silent: true,
+    output: '.',
+    branch: 'next',
```

Comment:
This will always generate alpha versions of Storybooks! I think it'd make sense to have `main` for latest and `next` for next